lib/list.js

```diff
@@ -0,0 +1,17 @@
+const wreck = require('wreck');
+const Boom = require('boom');
+
+module.exports = async function(email) {
```

Comment:
If this is going to be a fully functional method, then the arguments that are passed in should match the search requirements inside the list function, and not just `email` (I think you can search by `firstName` and `lastName` too? Maybe more?

Perhaps we should pass in a parameter object that we convert to param string instead of *just* email.